content/index.md

```diff
@@ -67,12 +67,12 @@ The source code for the GDL platform is released under an open sources license a
 
 ![alt text](/images/librarybooks.svg)
 
-### Libraries
+### GDL design
 
 </gridheader>
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+The GDL is based on Material Design, an open sources adaptable system of guidelines, components, and tools that support the best practices of user interface design. We have released our Sketch design files on GitHub.
```

Comment:
Hmm...synes denne teksten er litt i overkant lang for en beskrivelse for hva man finner på neste side.

Hva med f.eks:
```
The GDL is based on Material Design and tools that support the best practices of UI design. 
```

Setningen:
```We have released our Sketch design files on GitHub.``` synes jeg kan stå på neste side?

Reply:
Hei, da har jeg tatt inn deler av dine innspill. Jeg landet på at jeg fjernet det med github i siste setning men jeg har min opprinnelige første setning stå slik den gjorde. Jeg har også endre heading slik du foreslo.